### ISSUE TITLE
Some sites inaccessible, 502/403 errors

### ISSUE THREAD

**Issue by jonnym1** [barnacs/compy#38]
There are various sites I have found that cannot be accessed using this proxy with unhelpful error messages such as bad request or invalid reponse. I'm not a software developer so have had no luck tracking down what the issue is. If anyone gets 5 minuted can you look at
https://www.old-maps.co.uk/
or
https://www.realvnc.com/connect/download/vnc/windows/

Thanks, Jon.

**Comment by barnacs** [comment]
I can not reproduce this. Both sites load fine without errors for me 
through the proxy.
Are you using the latest version? Is your configuration correct?

**Comment by jonnym1** [comment]
Yep, thanks for pointing me in the right direction. Was using go 1.8 and upgraded to 1.10.1 and now working fine.
Thanks.
